src/index.js

```diff
@@ -155,6 +155,24 @@ function createFilteredErrorRule(filter: (CollectOutputElement) => any): (contex
   };
 }
 
+const MIN_COVERAGE_DIRECTIVE_COMMENT_PATTERN =
+  /(\s*eslint\s*['"]flowtype-errors\/enforce-min-coverage['"]\s*:\s*\[\s*(?:2|['"]error['"])\s*,\s*)(\d+)(\]\s*)/
```

Comment:
@amilajack I put a line break after `=`, but the pattern is still pretty long. AFAIK the only way to break this up further would be to use regexp constructor like this: https://stackoverflow.com/a/12317105/2696867

I personally prefer the regex literal, but lmk what you'd prefer

Reply:
This is fine, thanks! 😃